src/main/java/org/mule/extension/db/internal/result/resultset/IteratorResultSetHandler.java

```diff
@@ -23,10 +24,19 @@
 
   private final RowHandler rowHandler;
   private final StatementStreamingResultSetCloser streamingResultSetCloser;
+  private final Charset charset;
 
   public IteratorResultSetHandler(RowHandler rowHandler, StatementStreamingResultSetCloser streamingResultSetCloser) {
```

Comment:
Are we still using this constructor somewhere?

Reply:
No, we aren't